lib/accesscontrol/permitted_action_on_object_query.rb

```diff
@@ -21,8 +22,9 @@ def initialize(actor)
     #   # Can the user perform the action with id 5 for the Post with id 7?
     #   AccessControl::Records.can?(user, 5, Post, 7)
     def can?(action_id, object_type, object_id)
-      find_or_set_value(:can, @actor.class.name, @actor.id, action_id, object_type) do
+      find_or_set_value(:can, @segment_id, @actor.class.name, @actor.id, action_id, object_type) do
```

Comment:
I'm not sure we need to put `segment_id` in the cache key, do we? I think we only have it for faster DB lookups and really nothing else, right?

If that's the case, we can probably remove it from this line.

Reply:
Based on our discussion thread around caching earlier - I'm going to also remove this 👍